lib/solargraph/language_server/transport/adapter.rb

```diff
@@ -8,6 +8,20 @@ module Transport
       # A common module for running language servers in Backport.
       #
       module Adapter
+        # This runs in the context of Backport::Adapter, which
+        # provides write() - but if we didn't hide this behind a parse
+        # tag, it would override the one in the class.
+        #
+        # @!parse
+        #   # @abstract
+        #   # Write the change to the specified text.
+        #   #
+        #   # @param text [String] The text to be changed.
+        #   # @return [String] The updated text.
+        #   def write text
+        #     raise NotImplementedError
+        #   end
```

Comment:
You could also do this with a `@!method` directive:

```suggestion
        # @!method write(text)
        #   @abstract
        #   Write the change to the specified text.
        #   @param text [String] The text to be changed.
        #   @return [String] The updated text.
```

In this case they appear to be functionally equivalent, but sometimes `@!parse` can do unexpected things.

Reply:
Thanks!  Yeah, for sure I need to check my tag crib notes more instead of leaning on @!parse so much :)